server/liveblog/themes/themes_assets/classic/template.html

```diff
@@ -72,8 +83,42 @@ <h1 ng-if="timeline.settings.showTitle" ng-bind="timeline.blog.title"></h1>
                     <i class="fa fa-link fa-2"></i>
                 </a>
             </div>
+
             <div lb-item ng-repeat="item in post.items track by item._id" class="lb-item">
-                <div ng-switch="item.item_type">
+                <div ng-if="post.fullDetails && !$first" class="list-group-item">
+                    <div class="lb-timeline__header">
+                        <div class="lb-author" ng-if="timeline.settings.showAuthor">
+                            <img ng-if="timeline.settings.showAuthor && timeline.settings.showAuthorAvatar"
+                                 class="lb-author__avatar"
+                                 ng-class="{'lb-author__avatar': (pitem.item_type !== 'comment'),
+                                               'lb-author__comment': (item.item_type === 'comment')}"
+                                 ng-src="{{ item.original_creator.picture_url }}" />
+
+                            <div class="lb-author__date" ng-bind="(item.meta._created || item._created) | prettifyIsoDate"></div>
+                            <!-- author name, with or without mailto link  -->
+                            <div ng-if="!timeline.settings.authorNameLinksToEmail || post.mainItem.item_type === 'comment'"
+                                 class="lb-author__name"
+                                 ng-bind="post.mainItem.original_creator[timeline.settings.authorNameFormat]"></div>
+                            <div ng-if="timeline.settings.authorNameLinksToEmail && post.mainItem.item_type !== 'comment'"
+                                 class="lb-author__name">
+                                <a href="mailto:{{post.mainItem.original_creator.email}}"
+                                   ng-bind="post.mainItem.original_creator[timeline.settings.authorNameFormat]"></a>
+                            </div>
+                        </div>
+                        <div class="lb-item" ng-switch="item.item_type">
```

Comment:
There is too much code duplication here. I think that we can factorize it with `<script/>' templates or directives.